ui/src/components/Header/Controller.tsx

```diff
@@ -38,22 +41,27 @@ export const Controller = (): ReactElement => {
   }, [isLoading]);
 
   const buildMountArg = () => {
-    const OSPath = ddClient.host.platform === 'darwin'
-      ? 'Users'
-      : 'home';
+    if (user === ERROR_USER) {
```

Comment:
It seems like we're using a string here to encode that a user is not found. 
Isn't there a better way of facilitating this? We could also use '' to signify no user being found and for us to use the default volume dir